pvlib/bifacial/infinite_sheds.py

```diff
@@ -502,6 +504,9 @@ def get_irradiance(surface_tilt, surface_azimuth, solar_zenith, solar_azimuth,
       cells from the front surface. [W/m^2]
     - ``poa_front_ground_diffuse`` : ground-reflected diffuse irradiance
       reaching the module cells from the front surface. [W/m^2]
+    - ``shaded_fraction_front`` : fraction of row slant height from the
+      bottom that is shaded from direct irradiance on the front surface.
+      [unitless]
```

Comment:
```suggestion
    - ``shaded_fraction_front`` : fraction of the front surface,
      going up from the bottom, affected by row-to-row beam shading.
      [unitless]
```

Maybe this?  And likewise for `shaded_fraction_back`